pkg/backend/httpstate/policypack.go

```diff
@@ -321,16 +321,16 @@ func completeNodeJSInstall(finalDir string) error {
 }
 
 func completePythonInstall(finalDir, projPath string, proj *workspace.PolicyPackProject) error {
-	if err := python.InstallDependencies(finalDir, false /*showOutput*/, func(virtualenv string) error {
-		// Save project with venv info.
-		proj.Runtime.SetOption("virtualenv", virtualenv)
-		if err := proj.Save(projPath); err != nil {
-			return errors.Wrapf(err, "saving project at %s", projPath)
-		}
-		return nil
-	}); err != nil {
+	const venvDir = "venv"
+	if err := python.InstallDependencies(finalDir, venvDir, false /*showOutput*/); err != nil {
 		return err
 	}
 
+	// Save project with venv info.
+	proj.Runtime.SetOption("virtualenv", venvDir)
```

Comment:
now that this is a requirement - does this mean that the user must specify this in their Pulumi.yaml?

Reply:
It’s not a requirement. There isn’t any user-facing change with this modification.